getting-started.md

```diff
@@ -0,0 +1,142 @@
+
+> Tip: You can use the tab key to auto-fill the name of the file, after you type the first few characters.
+
+**MacOS & Linux**
+
+Open a new terminal. Type
+
+```console
+./go-wallet
+```
+to execute the program.
+
+**Windows**
+
+Open a new command prompt. Type
+
+```console
+go-wallet
+```
+to execute the program.
+
+> Tip: You can see a list of available commands by running  `./go-wallet -h` on MacOS and Linux, or `go-wallet -h` on Windows.
+
+### Create name and passphrase
+Next, **create a user name and passphrase** for your Wallet, and **create a public and private key** (genkey). 
+
+Replace "YOUR_CUSTOM_USERNAME" (below) with your chosen username:
+
+**MacOS & Linux**
+
+```console
+./go-wallet genkey -n "YOUR_CUSTOM_USERNAME"
+```
+
+**Windows**
+
+```console
+go-wallet genkey -n "YOUR_CUSTOM_USERNAME"
+```
+
+It will then prompt you to **input a passphrase**, and then **confirm that passphrase**. You'll use this username and passphrase to login to Vega Console. (Instructions on connecting to Console are below.)
+
+The genkey command in that instruction will generate public and private keys for the wallet, at the same time as creating a user name.
+
+You’ll see an output with a "mnemonic" and a public and private key. DO NOT SHARE YOUR MNEMONIC OR YOUR PRIVATE KEY.
+
+**The mnemonic acts as a backup key, from which the wallet can restore all your keys.** Keep it safe and secret. If lost, you won't be able to retrieve your keys. Anyone who has this mnemonic will be able to use your keys.
+
+You don’t need to save your private key, as you’ll be able to retrieve it from your Wallet in the future.
+
+> Tip: You can give each new key a nickname/alias. 
+> When creating a key, run 
+> 
+> MacOS & Linux: `./go-wallet genkey -name="YOUR_CUSTOM_USERNAME" --metas="name:CHOOSE_CUSTOM_ALIAS_FOR_KEY"`. 
+> 
+> Windows: `go-wallet genkey -name="YOUR_CUSTOM_USERNAME" --metas="name:CHOOSE_CUSTOM_ALIAS_FOR_KEY"`
+
+> Tip: To give an existing key a nickname/alias, run 
+> 
+> MacOS & Linux: `./go-wallet meta --metas="name:CHOOSE_CUSTOM_ALIAS_FOR_KEY" --name="YOUR_CUSTOM_USERNAME" --pubkey="REPLACE_THIS_WITH_YOUR_PUBLIC_KEY"`. 
+> 
+> Windows: `go-wallet meta --metas="name:CHOOSE_CUSTOM_ALIAS_FOR_KEY" --name="YOUR_CUSTOM_USERNAME" --pubkey="REPLACE_THIS_WITH_YOUR_PUBLIC_KEY"`
+
+> Tip: You can also use the meta command to tag a key with other data you might want, using a property name and a value. This will be useful for developing with Vega Wallet in the future.
+
+## Run the Wallet service
+Now, **connect your Wallet to the Testnet nodes**. The `init` command (below) will initialise the configuration. A configuration file will be stored in your home folder, in a folder called `.vega`.
```

Comment:
Is it still testnet ? I saw you updated to some documentation links to point toward fairground. Is it the same thing ?

Reply:
Even though docs and Console URLs have changed to fairground, the node URLs still say testnet, so I think it makes sense here to keep it called testnet.

Reply:
I split the difference and included a reference to Fairground, and hopefully I didn't just make it more confusing. What do you think?